.github/workflows/tests.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       - name: Install system dependencies
         run: sudo apt update && sudo apt install gettext mysql-client
       - name: Install Python dependencies
-        run: pip3 install -r src/requirements.txt -Ur src/requirements/dev.txt mysqlclient psycopg2-binary
+        run: pip3 install -r src/requirements.txt -Ur src/requirements/dev.txt mysqlclient psycopg2-binary==2.9.9
```

Comment:
You don't need to add a version to psyscopg2-binary since it is already pinned in `src/requirements/dev.txt` below